### PR TITLE
docs/set activation status

### PR DESCRIPTION
**Pull request by JGiter**
Added code example to activate solution

### CODE REVIEW

File: docs/activate_solution.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+```ts
+import { ApiPromise, WsProvider, Keyring } from "@polkadot/api";
+import { blake2AsHex } from "@polkadot/util-crypto";
+
+// Solution has status, which is used by registrar to enable voting round in this solution. Status
+// has three variants:
+// * Paused = 0. Solution not yet ready for voting
+// * Active = 1. New voting can be started
+// * Ended = 2. No new voting can be started
+// New solution has pending status. To change status `set_activation_status` extrinsic should be used.
```
Comment (knzeng-e) [inline docs/activate_solution.md]:
```suggestion
// New solution has Paused status. To change status `set_activation_status` extrinsic should be used.
```

**Review by jrhender — APPROVED** [review]
Thanks @JGiter

File: docs/activate_solution.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  });
+  const REGISTRAR_ADDRESS = REGISTRAR_KEYRING.address;
+
+  const solutionNamespace = "solution namespace";
+  await new Promise<void>(async (resolve) => {
+    let unsub = await api.tx.workerNodePallet
+      .setActivationStatus(
+        solutionNamespace,
+        1
+      )
```
Comment (knzeng-e) [inline docs/activate_solution.md:28]: What do think about the suggestion to make explicit the status we set in the example ?
```suggestion

  const solutionNamespace = "solution namespace";
  const solutionStatus = {
      paused: 0,
      active: 1,
      ended: 2,
  }
  await new Promise<void>(async (resolve) => {
    let unsub = await api.tx.workerNodePallet
      .setActivationStatus(
        solutionNamespace,
        solutionStatus.active
      )
```

**Review by knzeng-e — APPROVED** [review]
A few minor suggestions, but it looks good to me. Thank you @JGiter !

**Review by KaamilW — APPROVED** [review]

File: docs/activate_solution.md (hunk 2 above)
Comment (JGiter) [reply docs/activate_solution.md:28]: I don't see purpose of new variable. Meaning of statuses were described in comment

Comment (knzeng-e) [reply docs/activate_solution.md:28]: Some persons focus more on logic when reading a code, other more on context (and comments), so having this on both could be worth. But beside this "pedagogical" purpose, I agree with you, all information is available.

**Pull request merged**